docs/conf.py

```diff
@@ -1,13 +1,16 @@
-"""Sphinx configuration file for pytest-memray documentation."""
+"""Sphinx configuration file for pytest-memray documentation."""  # noqa: INP001
```

Comment:
Mm. It doesn't have any provision for handling scripts that aren't part of packages? That's annoying...

Reply:
All python files should be formatted, I don't see why this should be different.

Reply:
I think you misunderstood me. I'm not arguing that `docs/conf.py` shouldn't be formatted, I'm complaining that `ruff` thinks that `docs` is a Python package, and wants is to create a `docs/__init__.py` - it doesn't understand that `docs/conf.py` is a standalone script that's not part of any package.